geoapi/services/projects.py

```diff
@@ -62,65 +64,32 @@ def create(database_session, data: dict, user: User) -> Project:
         if user.tenant_id.upper() in custom_on_project_creation:
             custom_on_project_creation[user.tenant_id.upper()](database_session, user, project)
 
+        if project.system_id:
+            try:
+                system_users = get_system_users(database_session, user, project.system_id)
+                logger.info(f"Initial update of project_id:{project.id} system_id:{project.system_id} "
+                            f"system_path:{project.system_path} to have the following users: {system_users}")
+                users = [UserService.getOrCreateUser(database_session, u.username, project.tenant_id) for u in system_users]
+                project.users = users
+
+                if system_users:
+                    # Initialize the admin status
+                    users_dict = {u.username: u for u in system_users}
```

Comment:
Being pedantic here, but I feel like this line might read easier as `u for user in system_users`?

Reply:
updated in ca484df0879e4426c3f6e6f3b55adfe3ac6a9ff3